app/controllers/photoTagController.js

```diff
@@ -1,7 +1,42 @@
+import { PhotoTagCreate, PhotoTagResponse } from "../models/apiModels/photoTag.js";
+import PhotoTagAccessor from "../databaseAccessors/photoTagAccessor.js";
+import { ErrorRepeatedTagName, ErrorUnexpected, HttpError } from "../error/errors.js";
+
+
 /**
  * PhotoTagController Class
  *
  * This class controls the behaviour of any web request
  * related to PhotoTags
  */
-export default class PhotoTagController {}
+export default class PhotoTagController {
+  /**
+   * Creates a PhotoTag with ref fields populated
+   *
+   * @param {Request} req
+   * @param {Response} res
+   */
+  static async create(req, res) {
+    try {
+      const body = new PhotoTagCreate(req.body);
+      
+      // if creating a tag that already exists
+      const listOfTags = await PhotoTagAccessor.getAllTags();
+      if (listOfTags.map((tag) => tag.tagName).includes(body.tagName)) {
+        throw new ErrorRepeatedTagName();
+      } 
+      const creator = body.creatingUser; // assuming creatingUser is the userID
+      const newTag = await PhotoTagAccessor.createPhotoTag(body);
+      const updateNewTag = await PhotoTagAccessor.updatePhotoTag(newTag._id, creator);
```

Comment:
I'm not understanding these lines of code... you grab the `creatingUser` from the validated record (which now will be in the form of user email), you then create the record, with an incorrect creatingUser value, and then update the photoTag to include that same value again?

I would take a critical look at the flow in these lines again, and try to grasp exactly what these lines are accomplishing, if anything.